### PR TITLE
faq.rst: fix /etc/default/, SMTP CA truststore instructions

### PR DESCRIPTION
**Pull request by mchubby**

### CODE REVIEW

File: pages/faq.rst [hunk 1]
```diff
@@ -200,6 +200,10 @@ Outbound TLS connections have CA (*certification authority*) certificate verific
  
 This should be corrected by either adding the missing CA certificates to the Java default trust store (typically found at ``$JAVA_HOME/jre/lib/security/cacerts``), or a custom store that is configured (by using ``-Djavax.net.ssl.trustStore``) for the Graylog server process. The same procedure applies for both missing valid CAs and self-signed certificates.
 
+For Debian/Ubuntu-based systems using OpenJDK JRE, CA certificates may be added to the systemwide trust store. After installing the JRE (including ``ca-certificates-java``, ergo ``ca-certificates`` packages), place ``name-of-certificate-dot-crt`` (in PEM format) into ``/usr/local/share/ca-certificates/`` and run ``/usr/sbin/update-ca-certificates``. The hook script in ``/etc/ca-certificates/update.d/`` should automatically generate ``/etc/ssl/certs/java/cacerts``.
+
+Fedora/RHEL-based systems may refer to the `Fedora wiki article <https://fedoraproject.org/wiki/Features/SharedSystemCertificates>`
```
Comment (joschi) [inline pages/faq.rst]: Missing trailing underscore for the link reference, see http://www.sphinx-doc.org/en/stable/rest.html#hyperlinks

**Review by joschi — CHANGES_REQUESTED** [review]
@mchubby Thank you very much!

Except for the syntax issue with the hyperlink reference, this looks good to me.

**Comment by mchubby** [comment]
Thanks, done. I don't know how to squash the commits from the web ui, though. There should be a green button for squashing on the PR itself, though.

**Review by joschi — APPROVED** [review]

**Comment by joschi** [comment]
@mchubby Thanks!

**Pull request merged**